testkit/tests/currency/cryptocurrency.rs

```diff
@@ -219,7 +219,7 @@ impl CryptocurrencyApi {
         if let Some(wallet) = self.wallet(&public_key) {
             self.ok_response(&serde_json::to_value(wallet).unwrap())
         } else {
-            Err(IronError::new(ApiError::NotFound, (
+            Ok(Response::with((
```

Comment:
Yeah, now I understand why I don't like this code. There is more appropriate [way](https://docs.rs/exonum/0.5.1/exonum/api/trait.Api.html#method.not_found_response) to do that